persistence/msmq/subscription_config_core_[6,).partial.md

```diff
@@ -2,8 +2,24 @@ To configure MSMQ as the subscription persistence:
 
 snippet: ConfiguringMsmqPersistence
 
-By default NServiceBus uses a queue called `NServiceBus.Subscriptions`. If not specified otherwise, all endpoints will share that queue to store subscriptions. In order to specify a different queue, add the following `configSections` and subsequent config entry:
+By default NServiceBus uses a queue called `NServiceBus.Subscriptions`. If not specified otherwise, all endpoints will share that queue to store subscriptions. The following warning message will get logged:
 
-snippet: MsmqSubscriptionAppConfig
+> NServiceBus.Features.MsmqSubscriptionPersistence Could not find configuration section for Msmq Subscription Storage and no name was specified for this endpoint. Going to default the subscription queue
 
-WARNING: When using MSMQ Subscription Persistence on multiple endpoints running on the same machine, every endpoint requires a dedicated subscription storage queue.
+WARNING: When using MSMQ Subscription Persistence on multiple endpoints running on the same machine, every endpoint **must have** a dedicated subscription storage queue.
```

Comment:
@dvdstelt - multiple different seems to be redundant. Also, when you say, "Sharing a single queue will result in unexpected behavior" doesn't explain much.  How about:

If multiple endpoints are hosted on the same server, and if the endpoints are configured to use MSMQ subscription persistence, then each endpoint **must have** a dedicated subscription storage queue.